plugins/hydra_submitit_launcher/hydra_plugins/hydra_submitit_launcher/config.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+from dataclasses import dataclass
+from enum import Enum
+from typing import Optional
+
+from hydra.core.config_store import ConfigStore
+from hydra.types import ObjectConf
+
+
+class QueueType(Enum):
+    auto = "auto"
+    local = "local"
+    slurm = "slurm"
+
+
+@dataclass
+class SlurmQueueConf:
+    nodes: int = 1
+    gpus_per_node: int = 1
+    ntasks_per_node: int = 1
+    mem: str = "${hydra.launcher.mem_limit}GB"
+    cpus_per_task: int = 10
+    time: int = 60
+    partition: str = "dev"
```

Comment:
I am actually not sure that this makes sense for the opensource plugin version.
We should probably make FAIR specific.

@jrapin, what would be a good default value for the partition in the open source plugin (The FAIR cluster can get different default values through the fair-cluster-defaults plugin).

Reply:
set it to ```dev``` to be consistent with submitit's example https://github.com/facebookincubator/submitit/blob/master/docs/examples.md

Reply:
> set it to dev to be consistent with submitit's example

At some point we may remove the default since even dev probably does not exist everywhere... also using dev is not optimal as a default for FAIR :s no perfect solution here I think, dev is good enough for now :s